rwengine/src/objects/GameObject.cpp

```diff
@@ -23,12 +23,10 @@ void GameObject::setRotation(const glm::quat& orientation) {
 }
 
 float GameObject::getHeading() const {
-    auto hdg = glm::roll(getRotation());
-    return hdg / glm::pi<float>() * 180.f;
+    return glm::degrees(glm::roll(getRotation()));
```

Comment:
I think having a "hdg" (maybe rename to: `heading`) variable makes sense as it shows the intention / documents what goes on (extract heading -> convert to degrees). Without it, I'm somewhat lost reading that line.

Other instance in file too.

The use of `glm::degrees` looks better though :)